tests/docker/globus.debian11.Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM debian:11 
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=true
+
+#### install basic packages ####
+RUN apt-get update && apt-get install -y curl \
+    vim \
+    ftp \
+    telnet \
+    libcurl4-openssl-dev \
+    unzip \
+    python3 \
+    python3-distro \
+    python3-psutil \
+    python3-jsonschema \
+    python3-requests \
+    python3-pip \
+    python3-pyodbc
+
+#### Get and install iRODS repo ####
+RUN apt-get update && apt-get install -y wget gnupg2 lsb-release
+RUN wget -qO - https://packages.irods.org/irods-signing-key.asc | apt-key add -
+RUN echo "deb [arch=amd64] https://packages.irods.org/apt/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/renci-irods.list
+RUN apt-get update
+
+#### Install icommands - used to set up, validate and tear down tests. ####
+RUN apt-get install -y irods-icommands
+
+#### Install irods-dev, cmake, clang  - used to build the connector ####
+RUN apt-get install -y irods-externals-cmake3.21.4-0
+RUN apt-get install -y irods-externals-clang*
+RUN apt-get install -y irods-dev
+
+#### install basic packages ####
+RUN apt-get install -y curl \
+    g++ \
+    gcc \
+    vim \
+    ftp \
+    telnet \
+    libcurl4-gnutls-dev \
+    unzip \
+    python3 \
+    python3-distro \
+    python3-psutil \
+    python3-jsonschema \
+    python3-requests \
+    python3-pip \
+    python3-pyodbc
```

Comment:
Many / All of these packages could/should be installed along with the others in the first `RUN` step (on line 7).

Also, pretty sure we don't need `vim`.

Reply:
I install vim for my own purposes in case things fail.  Are we okay with leaving it in?

I will combine these steps.

Reply:
After talking with Alan we decided to go ahead and remove vim.  I'll just install it as needed for debugging purposes.